sass/cd/cd-header/_cd-language-switcher.scss

```diff
@@ -6,123 +6,182 @@
   position: relative;
 }
 
-// Language switcher button
+// Language Switcher button.
 .cd-language-switcher__btn {
   display: flex;
+  justify-content: flex-end;
   align-items: center;
   box-sizing: content-box;
   height: var(--cd-global-header-height);
-  padding: 0 0 0 15px;
+  padding: 0;
+  padding-inline: 1rem;
   line-height: var(--cd-global-header-height);
 
-  // Language switcher is always on the far edge of the global header.
-  // Cancel the border that we render for other elements.
-  &::after {
-    content: none;
-  }
-
-  // Place toggle indicator on edge of Global Header
-  .cd-icon--arrow-down {
-    min-height: 100%; // IE11 fix.
-    margin: auto; // Center vertically in macOS 10
-  }
-
   &:focus {
     outline: 3px solid var(--brand-primary--light);
   }
 
   @include tablet {
-    padding: 0 0 0 20px;
+    padding-inline: 1.5rem;
```

Comment:
Not to myself, should this be `padding-inline-start` to only apply to LTR left?

Reply:
Nope, just a change in styles. This matches the default styles which also apply padding to both inline directions.